src/Interfaces/ITagEngine.cs

```diff
@@ -6,15 +6,6 @@ namespace Geta.Tags.Interfaces
 {
     public interface ITagEngine
     {
-        PageDataCollection GetPagesByTag(string tagName);
```

Comment:
Mark methods as deprecated or obsolete?

Reply:
I am removing obsolete methods as those were living for too long in the codebase.